### ISSUE TITLE
[Due for payment 2026-03-30] Reports - Keyboard navigation does not work at the first/last item of Column list

### ISSUE THREAD

**Issue by jponikarchuk** [Expensify/App#85657]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.3.40-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** N/A - new feature, doesn't exist in prod
**If this was caught during regression testing, add the test name, ID and link from BrowserStack:** https://github.com/Expensify/App/pull/78869
**Email or phone of affected tester (no customers):** sjodinjsonjinodisjniod@gmail.com
**Issue reported by:** Applause Internal Team
**Bug source:** Exploratory - Significant User Experience Deterioration
**Device used:** Mac 26.2 / Chrome
**App Component:** Search

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to Reports > Expenses.
3. Click Columns.
4. Press Up arrow key.
5. Press Down arrow key until the last item in the list.
6. Press Down arrow key again.

## Expected Result:
- Pressing Up arrow key at the top of the list will focus on the last item in the list.
- Pressing Down arrow key at the last item of the list will focus on the first item in the list.

## Actual Result:
- Pressing Up arrow key at the top of the list does not navigate.
- Pressing Down arrow key at the last item of the list does not navigate.

## Workaround:
Unknown

## Platforms:
- [ ] Android: App
- [ ] Android: mWeb Chrome
- [ ] iOS: App
- [ ] iOS: mWeb Safari
- [ ] iOS: mWeb Chrome
- [x] Windows: Chrome
- [x] MacOS: Chrome / Safari

## Screenshots/Videos

https://github.com/user-attachments/assets/28c7adc3-a9ff-40aa-a69b-5b08ae9e5680

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by TaduJR** [comment]
@mallenexpensify 

> @TaduJR @aimane-chnaif were either of you sent offers in Upwork?

No Automatic offer sent on my side

Payment is not needed here.

**Comment by mallenexpensify** [comment]
ugggh, forgot to reduce pay in the parent issue for this regression, commented [there](https://github.com/Expensify/App/issues/78188#issuecomment-4166576326)
